packages/docusaurus-theme-classic/src/theme/PaginatorNavLink/index.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import Link from '@docusaurus/Link';
+import type {Props} from '@theme/PaginatorNavLink';
+
+function PaginatorNavLink(props: Props): JSX.Element {
```

Comment:
not a big deal for now, but this is bad encapsulation that Infima is using a `--next` suffix in parent class to determine the position/direction of the arrow. This should be a props instead of a parent class, and this component should be self-sufficient to render properly without using any parent CSS